client/src/components/wishlistIcon/WishlistIcon.js

```diff
@@ -9,15 +9,17 @@ import { selectSeriesWishlistStatus } from "../../features/wishlist/wishlist.sel
 
 const WishlistIcon = (props) => {
     const dispatch = useDispatch();
-    const { relatedUser, series, className } = props;
+    const { relatedUser, series, className, disableClick = false } = props;
 
     const wishlistFillStatus = useSelector((state) => selectSeriesWishlistStatus(state, series._id));
+    
+    const classes= `${className} ${disableClick ? " pe-none" : ""}`;
 
     const updateUserWatchlist = async (event) => {
         event.preventDefault();
 
         try {
```

Comment:
Add `if (!disableClick)` because it'll affect your store if you click on an icon that doesn't belong to your user